utils/airdropWindows.ts

```diff
@@ -38,27 +37,29 @@ export enum WindowStatus {
 //     return new Date(AregistrationStart).getTime() - new Date(BregistrationEnd).getTime();
 //   });
 
-
 export const findActiveWindow = (
-  windows: AirdropWindow[]
+  windows: AirdropWindow[],
 ): AirdropWindow | undefined => {
   const sortedWindows = windows
     .slice()
     .sort(
       (windowA, windowB) =>
-        windowA.airdrop_window_id - windowB.airdrop_window_id
+        windowA.airdrop_window_id - windowB.airdrop_window_id,
     );
 
   const activeWindow = sortedWindows.find((windowA) => (
-      checkDateIsGreaterThan(windowA.airdrop_window_registration_start_period, moment()) ||
+    checkDateIsGreaterThan(windowA.airdrop_window_registration_start_period, moment()) ||
```

Comment:
@raamb assign `moment()` to a variable to avoid unnecessary initialization.

Reply:
Next PR has this and all other date fixes